### PR TITLE
About page

### PR DESCRIPTION
**Pull request by marchingkoala**
- cleaned up
- added instructions with images

### CODE REVIEW

File: src/components/About/AboutPage.tsx [hunk 1]
```diff
@@ -3,11 +3,48 @@ import "./about.css";
...
-        meal of the day based on ingredients in your fridge!{" "}
-      </p>
+      <p className="about_p">Chef's Kiss is a kitchen helper application that can help you decide
+        meal of the day based on ingredients in your fridge!</p>
+      <div className="withAlexa">
+        <h2 className="aboutAlexaH2">How to use with Alexa</h2>
+        <div className="AboutalexaContainer">
+          <div className="AboutAlexbox">
+            <img src="../static/instructionIcon-01.jpg" width="220px" />
+            <p className="instruction_p">Activate by saying "Alexa, open Chef's Kiss</p>
```
Comment (emccormick16) [inline src/components/About/AboutPage.tsx:13]: Add a quote after Kiss

File: src/components/About/AboutPage.tsx [hunk 2]
```diff
@@ -3,11 +3,48 @@ import "./about.css";
 const AboutPage = () => {
   return (
     <div className="about">
-      <h2 className="about_h">About Page</h2>
-      <p className="about_p">
-        Chef's Kiss is a kitchen helper application that can help you decide
-        meal of the day based on ingredients in your fridge!{" "}
-      </p>
+      <p className="about_p">Chef's Kiss is a kitchen helper application that can help you decide
+        meal of the day based on ingredients in your fridge!</p>
```
Comment (emccormick16) [inline src/components/About/AboutPage.tsx:7]: I would probably rephrase this to say "that can help you decide what to make based on the ingredients in your fridge". Not a necessary change though

File: src/components/About/AboutPage.tsx (hunk 1 above)
Comment (marchingkoala) [reply src/components/About/AboutPage.tsx:13]: done

File: src/components/About/AboutPage.tsx (hunk 2 above)
Comment (marchingkoala) [reply src/components/About/AboutPage.tsx:7]: done!

**Review by emccormick16 — APPROVED** [review]
